src/main/java/org/kuali/kfs/pdp/businessobject/PaymentDetail.java

```diff
@@ -137,6 +137,18 @@ public boolean isDisbursementActionAllowed() {
     }
 
     /**
+     * Wrapper for addNote, loops through the list of notes passed in and calls that.
+     *
+     * @param pnts list of notes to add
+     */
+    public void addNotes(List<PaymentNoteText> pnts) {
+        pnts.stream().forEach(n -> {
```

Comment:
perhaps a better variable name, paymentNoteTextes?

Reply:
This is an overlay of a base KFS file, and this change just makes the method conform to how it appears in the base PaymentDetail file. If you think it could still use cleanup, then I'm fine with renaming the variable.

Reply:
lets leave as is